pkg/agent/cniserver/pod_configuration.go

```diff
@@ -141,6 +143,10 @@ func BuildOVSPortExternalIDs(containerConfig *interfacestore.InterfaceConfig) ma
 	externalIDs[ovsExternalIDIP] = getContainerIPsString(containerConfig.IPs)
 	externalIDs[ovsExternalIDPodName] = containerConfig.PodName
 	externalIDs[ovsExternalIDPodNamespace] = containerConfig.PodNamespace
+	if containerConfig.IFDev != defaultIFDevName {
+		// Save interface name for a secondary interface.
+		externalIDs[ovsExternalIDIFDev] = containerConfig.IFDev
+	}
```

Comment:
would there be any harm in storing it even if it is the default / eth0?

Reply:
I want not to change the existing behavior esp. the interface name is useless for primary interfaces. Also for upgrade case, primary interface name will be missing for existing interfaces, even no harm either.

Reply:
No strong preference from me there